draft-ietf-quic-multipath.md

```diff
@@ -1416,34 +1398,46 @@ Sequence Number:
 
 ## MAX_PATHS frames {#max-paths-frame}
 
-A MAX_PATHS frame (type=0x15228c0b) informs the peer of the cumulative number of paths
-it is permitted to open.
+A MAX_PATHS frame (type=0x15228c0b) informs the peer of largest Path ID that
+is permitted to use on any path.
 
 MAX_PATHS frames are formatted as shown in {{fig-max-paths-frame-format}}.
 
 ~~~
 MAX_PATHS Frame {
   Type (i) = 0x15228c0b,
-  Maximum Paths (i),
+  Maximum Path Identifier (i),
 }
 ~~~
 {: #fig-max-paths-frame-format title="MAX_PATHS Frame Format"}
 
 MAX_PATHS frames contain the following field:
 
 Maximum Path Identifier:
-: A count of the cumulative number of path that can be opened
-  over the lifetime of the connection. This value cannot exceed 2^32-1, as it is not
-  possible to encode Path IDs larger than 2^32-1. Receipt of a frame that permits
-  opening of a path with Path ID larger than this limit MUST be treated
-  as a connection error of type FRAME_ENCODING_ERROR.
+: The largest Path ID that can be used for this connection.
+  This value cannot exceed 2^32-1, as it is not
+  possible to encode Path IDs larger than 2^32-1 as variable integer.
```

Comment:
```suggestion
  secure to use Path IDs larger than 2^32-1 with the security considerations outlined in {{multipath-aead}}.
```

Reply:
@huitema is right. Variable integer is able to encode up to 2^62 -1. 
Using the Path ID as part of the AEAD nonce input limit the upper bound of Path ID.